config/locales/en.yml

```diff
@@ -3313,18 +3313,17 @@ en:
           title_html: Please review your retirement income (1099-R) details for <strong>%{payer_name}</strong>.
       return_status:
         accepted:
-          check_payable: Make your check or money order payable to Arizona Department of Revenue. Write your full SSN, “%{filing_year} Tax” and 140 on your payment.
-          check_your_refund_html: <a target="_blank" rel="noopener nofollow" href="%{tax_refund_url}">Check on the status</a> of your refund.
+          check_payable: Make your check or money order payable to %{payable_to}. Write your full SSN or ITIN, “%{filing_year} Tax”, and "Form %{form_number}" on your payment.
           direct_debit_html: If you have a remaining tax balance, remember to <strong>pay your taxes online at <a target="_blank" rel="noopener nofollow" href="https://%{tax_payment_url}">%{tax_payment_url}</a></strong> or by mail before the April 15 filing deadline.
-          download_title: Download a copy of your %{filing_year} %{state_name} state tax return until December 31, %{filing_year + 1}
+          download_title: Download a copy of your %{year} %{state_name} state tax return until December 31, %{year}
```

Comment:
I know the  "+ 1" breaks here, but this message does need to show different years (e.g. "Download a copy of your 2024 AZ state tax return until December 31, 2025")